static/js/app.js

```diff
@@ -1,14 +1,25 @@
 (function() {
+  // A bunch of code expects a global "State" object. Set it here if not 
+  // set yet.
+  if (!window.State) {
```

Comment:
We do add this in the script block of each template that needs it. But it might be a good idea to add it there, for those templates that don't receive window.State directly from the template.